### ISSUE TITLE
Contrast Issues in links

### ISSUE THREAD

**Issue by ayanchoudhary** [cc-archive/vocabulary-legacy#454]
## Problem

There is an issue with the contrast of links especially tomato over white.
Currently, these links have a contrast ratio of **3.45** whereas the desired minimum contrast ratio should be **4.5** or **AA/AAA large**

## Additional context

![CC Search - Brave 12-06-2020 15_19_06](https://user-images.githubusercontent.com/45792768/84490412-c52d4c80-acc0-11ea-9cec-d3b932cbf5cd.png)

![CC Search - Brave 12-06-2020 15_24_32](https://user-images.githubusercontent.com/45792768/84490671-1a695e00-acc1-11ea-98ed-0c1b4779d54f.png)

![CC Search - Brave 12-06-2020 15_38_22 (2)](https://user-images.githubusercontent.com/45792768/84492163-60bfbc80-acc3-11ea-8e4a-a228585baf8e.png)

## Implementation

- [ ] I would be interested in implementing this feature.

**Comment by ayanchoudhary** [comment]
@panchovm @annatuma

**Comment by Atanda1** [comment]
I'm interested in fixing this issue

**Comment by Atanda1** [comment]
I'll just go ahead

**Comment by Atanda1** [comment]
Do you think we should just increase the font rather than changing the brand color

**Comment by fcoveram** [comment]
Changing from normal to large size is a tremendous increment. It will difficult lecture because every section will use a big portion of the layout. For now, we are working on this new palette.

**Comment by Aakash2408** [comment]
Hey @panchovm  @kgodey  @zackkrida Is this issue still open ?

**Comment by kgodey** [comment]
This is currently blocked.
